app/controllers/album.js

```diff
@@ -1,11 +1,13 @@
-const albumsService = require('../services/albums');
+const { getAlbums, getPhotosBy, albumRegister, findAlbumBy } = require('../services/albums');
 const logger = require('../logger');
 const message = require('../constants');
+const { decodedToken } = require('../utils');
+const { albumMapper } = require('../mappers/mappers');
+const errors = require('../errors');
 
-exports.getAlbums = (req, res, next) => {
+exports.getAllAlbums = (req, res, next) => {
```

Comment:
its ok to call getAlbums in order to make the endpoint more generic. you may want to get all the albums sometimes but other times you may only want some albums passing a filter.
it wont be good to create two different endpoints for doing that